core/trino-main/src/main/java/io/trino/operator/StreamingAggregationOperator.java

```diff
@@ -275,6 +275,7 @@ private void updateMemoryUsage()
             else {
                 userMemoryContext.setBytes(memorySize);
             }
+            return true;
```

Comment:
add a comment that this method returns boolean to match `UpdateMemory` interface

Reply:
Done